server/routes/api/place.py

```diff
@@ -137,7 +140,7 @@ def child(dcid):
     for place_type in child_places:
         child_places[place_type].sort(key=lambda x: x['pop'], reverse=True)
         child_places[place_type] = child_places[place_type][:CHILD_PLACE_LIMIT]
-    return json.dumps(child_places)
+    return Response(json.dumps(child_places), 200, mimetype='application/json')
```

Comment:
It doesn't work for one of these handlers which is being called by another function. Please check that all the handlers still return correctly in the chrome network inspector.

Reply:
good to know. Checked a few pages and urls, these changes are okay.